doc/input-output-reference/src/overview/group-design-objects.tex

```diff
@@ -891,7 +891,7 @@ \subsubsection{Inputs}\label{inputs-012}
 
 This field is the name of schedule that defines how outdoor air requirements change over time. The field is optional. If left blank, the schedule defaults to 1.0. If used, then the schedule values are multiplied by the outdoor air flow rate defined by the previous fields. The schedule values must be between 0 and 1, inclusive.  
 
-If this DesignSpecification:OutdoorAir object is referenced by a Controller:MechanicalVentilation object (either directly or indirectly through Sizing:Zone), the schedule will be applied to all types of outdoor air calculations for the corresponding zone, regardless of the System Outdoor Air Method selected. If the schedule value is zero, then the zone will be completely removed from the system outdoor air calcaulations.
+If this DesignSpecification:OutdoorAir object is referenced by a Controller:MechanicalVentilation object (either directly or indirectly through Sizing:Zone), the schedule will be applied to all types of outdoor air calculations for the corresponding zone, regardless of the System Outdoor Air Method selected. If the schedule value is zero, then the zone will be completely removed from the system outdoor air calculations.
 
```

Comment:
I was not asking about Terminal vs OA Controller.  Rather, within Controller:MechanicalVentilation, if the system method is, say, Sum, but the DesignSpec:OA object that is referenced in the Controller;MechanicalVentilation object says ProportionalControlBasedonOccupancySchedule, then what method will be used by the Controller:MechanicalVentilation object to calculate the min OA flow?  I think the system method will be used.  Yes, that's what the code shows in [MixedAir::VentilationMechanicalProps::CalcMechVentController](https://github.com/NREL/EnergyPlus/blob/fdb3cf27a3f041101f9cbbceab78a7bcfc0c89e3/src/EnergyPlus/MixedAir.cc#L3685). The method there is controlled by `this->SystemOAMethod`.  I don't know if we should mention that here or not.